pkg/controlplane/instance.go

```diff
@@ -636,6 +637,8 @@ var (
 		flowcontrolv1beta1.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // remove in 1.26
 		flowcontrolv1beta2.SchemeGroupVersion.WithResource("flowschemas"),                 // remove in 1.29
 		flowcontrolv1beta2.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // remove in 1.29
+		flowcontrolv1beta3.SchemeGroupVersion.WithResource("flowschemas"),
+		flowcontrolv1beta3.SchemeGroupVersion.WithResource("prioritylevelconfigurations"),
```

Comment:
Could we add the life cycle comment here as well?

Reply:
I added the comment, but can we actually remove `v1beta*` since these are legacy?

Reply:
done